gaia-switch.js

```diff
@@ -44,11 +44,31 @@ module.exports = component.register('gaia-switch', {
     this.disabled = this.getAttribute('disabled');
     this.checked = this.getAttribute('checked');
 
+    // put non-critical operations in post-load time frame
+    // to save loadtime.
```

Comment:
Yes. 

And for a11y related actions, they do affect a part of impairable user's interaction. But as above mentioned, the `50ms`+ post-load idle time can also be utilized to serve that purpose.